src/main/java/Duke.java

```diff
@@ -1,10 +1,57 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "____________________________________________________________\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "____________________________________________________________\n";
+
+        System.out.println(logo);
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        Task[] todo = new Task[100];
+        int todo_index = 0;
+        while (true) {
+            if (input.substring(0, 4).equals("done")) {
+                int i = Integer.parseInt(input.substring(5));
+                todo[i - 1].markAsDone();
+                System.out.println("____________________________________________________________");
+                System.out.println("Nice! I've marked this task as done: ");
+                System.out.println(todo[i - 1].toString());
+                System.out.println("____________________________________________________________");
+            } else {
+                switch (input) {
+                case "list":
+                    System.out.println("____________________________________________________________");
```

Comment:
Maybe a print function could be added to print out the lines instead of adding the lines each time you print?